src/components/frame/v5/pages/UserPreferencesPage/UserPreferencesPage.tsx

```diff
@@ -23,9 +23,18 @@ const UserPreferencesPage: FC<UserPreferencesPageProps> = ({
 }) => {
   const { user } = useCanEditProfile();
   const isMobile = useMobile();
+  const [isCopied, setIsCopied] = useState(false);
```

Comment:
is not necessary, you can use `isCopied` from `useCopyToClipboard` hook. If you need change setTimeout delay please pass prop with number to hook